src/Sentry/Laravel/Features/ConsoleSchedulingIntegration.php

```diff
@@ -17,6 +18,8 @@
 
 class ConsoleSchedulingIntegration extends Feature
 {
+    private $cacheStore = null;
```

Comment:
My only suggestion, otherwise this looks good to me!

```suggestion
	/*
	 * @var string|null
	 */
    private $cacheStore = null;
```

Reply:
Thanks, added!